tests/test_auto_annotator.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+test_auto_annotator
+----------------------------------
+
+Tests for `auto_annotator` module
+"""
+import pytest
+
+from mindmeld.auto_annotator import SpacyAnnotator
+
+SA = SpacyAnnotator("")
```

Comment:
you should configure this as a fixture object

is that okay not to pass in app path? you can pass in app path for either kwik e mart or home assistant

Reply:
Updated, used "kwik_e_mart"